Segment fault in libOpenflow for Traceflow to an external IP

**Describe the bug**
Got a segment fault for a Traceflow to the Node IP, or the gw0 IP, or an external IP that is reachable from the source.

It is probably triggered by the returned Traceflow packets from antrea-gw0.

**To Reproduce**
For example, antctl traceflow -S kube-system/coredns-74ff55c5b-6sz9g -D nodeIP

**Versions:**
Please provide the following information:
 - Antrea version: 0.13

**Additional context**
The agent log and backtrace:

```
I0217 05:08:27.601151    3511 traceflow_controller.go:174] Processing Traceflow kube-system-coredns-74ff55c5b-6sz9g-to-172.100.1.1-6w88m86n ADD event
I0217 05:08:27.605085    3511 traceflow_controller.go:180] Processing Traceflow kube-system-coredns-74ff55c5b-6sz9g-to-172.100.1.1-6w88m86n UPDATE event
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x155139b]

goroutine 150 [running]:
github.com/contiv/libOpenflow/openflow13.DecodeMatchField(0x4210001, 0xc00057e846, 0x66, 0x7ba, 0xc000a4a000, 0xc0003c4900, 0xc0, 0xc000a4a000)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/openflow13/match.go:452 +0xc1b
github.com/contiv/libOpenflow/openflow13.(*MatchField).UnmarshalBinary(0xc000c69d30, 0xc00057e842, 0x6a, 0x7be, 0x5, 0xc000a4a000)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/openflow13/match.go:178 +0x13d
github.com/contiv/libOpenflow/openflow13.(*Match).UnmarshalBinary(0xc000300298, 0xc00057e818, 0x94, 0x7e8, 0x0, 0x0)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/openflow13/match.go:92 +0x150
github.com/contiv/libOpenflow/openflow13.(*PacketIn).UnmarshalBinary(0xc000300280, 0xc00057e800, 0xac, 0x800, 0xc0006bc738, 0x1)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/openflow13/openflow13.go:364 +0x165
github.com/contiv/libOpenflow/openflow13.Parse(0xc00057e800, 0xac, 0x800, 0x4074f5, 0xc000564000, 0xc0006bc738, 0x1)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/openflow13/openflow13.go:145 +0x625
github.com/contiv/ofnet/ofctrl.(*Controller).Parse(0xc0004ba870, 0xc00057e800, 0xac, 0x800, 0x2154501, 0xc0007de160, 0x0, 0x0)
        /home/shenj/go/pkg/mod/github.com/wenyingd/ofnet@v0.0.0-20210205051801-5a4f247248d4/ofctrl/ofctrl.go:323 +0x54
github.com/contiv/libOpenflow/util.(*MessageStream).parse(0xc0005641e0)
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/util/stream.go:158 +0x1cb
created by github.com/contiv/libOpenflow/util.NewMessageStream
        /home/shenj/go/pkg/mod/github.com/contiv/lib!openflow@v0.0.0-20201116154255-01db743640b1/util/stream.go:72 +0x1e9
```

--- a/go.mod
+++ b/go.mod
@@ -70,6 +70,9 @@ replace (
 	// hcshim repo is modifed to add "AdditionalParams" field to HNSEndpoint struct.
 	// We will use this replace before pushing the change to hcshim upstream repo.
 	github.com/Microsoft/hcsshim v0.8.9 => github.com/ruicao93/hcsshim v0.8.10-0.20210114035434-63fe00c1b9aa
+	// temporary replacement to avoid Antrea Agent panics for some Traceflow requests
+	// see https://github.com/vmware-tanzu/antrea/issues/1878
+	github.com/contiv/libOpenflow => github.com/antoninbas/libOpenflow v0.0.0-20210218001059-32f2e57d0435
 	// antrea/plugins/octant/go.mod also has this replacement since replace statement in dependencies
 	// were ignored. We need to change antrea/plugins/octant/go.mod if there is any change here.
 	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20210205051801-5a4f247248d4

